Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -172,8 +168,18 @@ def urlscan_submit_url():
 
 
 def format_results(uuid):
+    # Scan Lists sometimes returns empty
+    num_of_attempts = 0
     response = urlscan_submit_request(uuid)
-    demisto.debug('urlscan_submit_request response: {}'.format(response))
+    scan_lists = response.get('lists')
+    while scan_lists is None:
```

Comment:
please add condition for `num_of_attempts` so we wont get into endless loop here